filament/src/materials/blitHigh.mat

```diff
@@ -27,20 +27,20 @@ vertex {
 }
 
 fragment {
+    void dummy() {}
+
     void postProcess(inout PostProcessInputs postProcess) {
         highp vec2 uv = variable_vertex.xy; // interpolated to pixel center
-        highp float du = materialParams.resolution.z;
-        highp float dv = materialParams.resolution.w;
         vec4 c0, c1;
-        c0  = textureLod(materialParams_color, uv + vec2(-du, -dv), 0.0);
-        c0 += textureLod(materialParams_color, uv + vec2( du, -dv), 0.0);
-        c0 += textureLod(materialParams_color, uv + vec2( du,  dv), 0.0);
-        c0 += textureLod(materialParams_color, uv + vec2(-du,  dv), 0.0);
+        c0  = textureLodOffset(materialParams_color, uv, 0.0, ivec2(-1, -1));
```

Comment:
So one thing I'm wondering is whether textureLodOffset() does the equivalent of a textureSize() per call. Did you measure before/after?

Reply:
I did a quick test with bloom, it doesn't seem to make any difference (1ms @ 425MHz / 1080p / P4)